src/api/server.rs

```diff
@@ -23,7 +23,7 @@ impl Server {
 
         let (_shutdown, rx) = oneshot::channel();
         let (addr, server) = warp::serve(routes).bind_with_graceful_shutdown(
-            config.api.bind.expect("Invalid socket address"),
+            config.api.address.expect("Invalid socket address"),
```

Comment:
I realize this was already like this, but it seems like `config.api.address` is an `Option`? Would `.expect("No socket address")` be better here?

I'm imagining this is already validated before it gets to this point?

Reply:
Yeah, you're right. The perils of `Option<T>` and `Result<T>` sharing an unwrap API 😄 Well spotted 👍 Fixed.